sensor/kubernetes/certrefresh/certrepo/service_certificates_repository.go

```diff
@@ -185,6 +185,9 @@ func (r *ServiceCertificatesRepoSecrets) patchServiceCertificate(ctx context.Con
 		Op:    "replace",
 		Path:  "/data",
 		Value: r.secretDataForCertificate(secretSpec, caPem, cert),
+	}, {Op: "replace",
+		Path:  "/metadata/labels",
+		Value: map[string]string{"rhacs.redhat.com/tls": "true"},
```

Comment:
I think this should be 
```suggestion
	}, {Op: "replace",
		Path:  "/metadata/labels",
		Value: utils.GetTLSSecretLabels(),
```